server/odc-service/src/main/java/com/oceanbase/odc/service/datatransfer/task/TransferTask.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.service.datatransfer.task;
+
+import java.util.List;
+
+import com.oceanbase.odc.service.datatransfer.model.DataTransferScope;
+import com.oceanbase.tools.loaddump.common.model.ObjectStatus;
+
+public interface TransferTask {
+
+    void init();
+
+    void transfer();
```

Comment:
what is the input and output of this method?
if you are a user of this interface, how can you know how to use this method?(If I call this method, where does `TransferTask` get input from and where will it output the export results? )